theano/d3viz/__init__.py

```diff
@@ -2,8 +2,10 @@
 
 has_requirements = False
 try:
-    import pydot as pd
-    if pd.find_graphviz():
-        has_requirements = True
+    # pydot-ng is a fork of pydot that is better maintained
+    import pydot_ng as pd
 except ImportError:
-    pass
+    # fall back on pydot if necessary
+    import pydot as pd
```

Comment:
I think I just found the problem. pydot is an optional dependency. So this one can fail and need to be in another try...except. This is what make python 3 tests fail, as it don't have pydot of installed.

Also, I grepped in Theano and this version of the import isn't used. d3viz use the one in the file d3viz/formatting.py. So you can just delete the import of pydot from this file.